Fig 3:
1. Add ‘ns’ bar for Stachelek T vs McEvoy below the other bars.  
2. Move the asterisks a little closer to bar they relate to; at present the lower asterisk seems closer to the higher bar. The distance seems about right for ‘ns’ so the asterisks may appear high if the font wants to make them appear as a superscript.  
3. Font size for Study Authors can be slightly increased as long as labels won’t run together.  
4. “A” and “B” panel labels can be larger to stand out more.
 
Fig. 4:
1. In A, light boxes around C11orf65 & ATM and MFRP & C1QTNF5
2. In B, DUOXA2 and DUOXA1 are new genes but seem to be the same gene with two names (as for RBM) given the variant effect is the same; suggest to review why this variant (and others, if there are any) suddenly appears (new genome annotation?), and if it’s correct, revise variant numbers in the text and other figures, novel recurrences, and ontologies accordingly.
3. Can use same line spacing in “B” as in “A”?  
4. Y-axis labels “Protein Consequence” as two words without :   moved a bit to left to separate from the gene names
5. Optional, move the keys left to fit within the x-axes, but still aligned with each other. The panel A key can move up a bit to not crowd the data.
 
Fig. S3
1. A few without X appear to be dropped in Fig. 4
2. In S3A:  SLC7A3:p.Glu222Lys, HYDIN:p.Ala4026Thr,  ZER1 and all above it.
3. In S3B:  PABPN1L:p.Arg169Cys, ZNF41:p.Asn171Lys, CYB561D1:p.Glu70Val, ALKBH4:p.Glu161Lys
4. Looks like we should retain PABPN1L,  ZNF41, and SLC7A3 because each VAF = ~1.00 in RB41 T and CL and there’s low VAF only in RB24 T, suggesting possible contamination.  
5. There’s a similar pairing for RB41 CL – RB24T  for MFRP & C1QTNF5, SPRED, MYCT, and CDK13, all of which were retained in the RB46 CL, which I think nails down it’s contamination.  
6. To be consistent we need to add back PABPN1L,  ZNF41, and SLC7A3 to the RB41 T and CL variants and adjust other figures and text accordingly.  
7. After submission we should Sanger sequence these (maybe HYDIN) in RB41 T or CL to confirm as careful reviewer would ask for it.
 
Fig. 5
1. In B, the size of each block can be same as in A.
2. Consider to not exclude McEvoy, not clear it’s valid to include from two different targeted sequencing unless the genes examined in each are the same; e.g., McEvoy examined BCOR and MYCN but not ARID1A and otherwise didn’t find anything new. 
3. Can adjust the % to the Afshar total sample number. Best if A and B can use same Alterations and Study colors, not critical.
 
Fig. 6. 
1. Still surprised covalent chromatin FDR decreases so much when it increased in prior version.  Can compare and re-run the two data inputs?  Was this done with 12 BCOR?
2. Consider to move dashed line down to 1.
3. Minor: Center and reduce font of the key, re-adjust positions of ontology descriptions.
 
S5 – Not sure we need it but if included we need larger fonts and combine in a single panel.  Why show three SAMD9 variants, one NAF1, and two PAN2?
 
Haven’t gone through the Tables. Some may need to revise based on issues above. 
